steamapi.go

```diff
@@ -38,9 +41,10 @@ func (s SteamMethod) Request(data url.Values, v interface{}) error {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
-		return errors.New(fmt.Sprintf("Status code %d returned by %v", resp.StatusCode, string(s)))
+		return fmt.Errorf("steamapi %s Status code %d", s, resp.StatusCode)
 	}
 
 	d := json.NewDecoder(resp.Body)
-	return d.Decode(&v)
+	_ = d.Decode(&v)
```

Comment:
I don't like ignoring errors either. I must admit I did that to quickly integrate and I didn't have much go experience at that time. I'm a bit afraid of side effects on my side if we handle the json decode error, BUT I guess that is for the best.
Steam unfortunately doesn't use the HTTP error codes to return their errors. So you have a 200 status with HTML inside :/.

Let's return the error then, I'll monitor any potential issue on my side.